datapreview.tex

```diff
@@ -1,19 +1,20 @@
 \section{Data Previews and Data Release 1} 
 \label{sec:datapreview}
 
-A series of three Data Previews (DP) are planned  based on commissioning data to support the community as they develop their LSST analysis software and worfklows, and to enable high-impact science as soon as possible.
+A series of three Data Previews (DP) are planned to support the community as they develop their LSST analysis software and workflows, and to enable high-impact science as soon as possible.
 \begin{itemize}
 \item Data Preview 0 (DP0): Based on simulated LSST-like data.
-\item Data Preview 1 (DP1): Based on a subset of early science-grade commissioning data taken with LSSTCam.
+\item Data Preview 1 (DP1): Based on a subset of early science-grade commissioning data taken with either ComCam or LSSTCam.
 \item Data Preview 2 (DP2): Based on a full reprocessing of all science-grade LSSTCam data taken during commissioning.
 \end{itemize}
 
-Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), these Data Previews will necessarily be limited in their area and temporal coverage relative to full a Data Release, however all Data Preview data products will be in the same science data model format as for future Data Releases.
+As both DP1 and DP2 will be based on commissioning data, they will not be released until after at least the 30-day embargo period for commissioning data has passed.
```

Comment:
Suggest removing this. In all cases DPs take longer than 30 days to process and provide to community

Reply:
Maybe it could be reworded, but if it isn't mentioned at all in this document, it's not clear where a community reader of this document would learn about the embargo policy.  Are we expecting a reader to have read DMTN-199 independently?  I think RTN-011 is quite a bit higher up the stack of what people are likely to read.

Reply:
Or are you saying that, if we explicitly state latencies for DP1 and DP2 that are more than 30 days from the **last** data taken that's included in the DP, the embargo policy is simply irrelevant?  In that case perhaps we should say something like "No commissioning data will be released to data rights holders other than through these data previews"?

Reply:
The last statement is true in any case -- "No commissioning data will be released to data rights holders other than through these data previews".

Reply:
Reworked to simplify and include one reference to DMTN-199 covering all data products